reconcile/utils/gql.py

```diff
@@ -129,17 +125,11 @@ def query(self, query, variables=None, skip_validation=False):
 
         if self.validate_schemas and not skip_validation:
             forbidden_schemas = [
-                schema for schema in query_schemas if schema not in self._valid_schemas
+                schema for schema in query_schemas if schema not in self._valid_schemas  # type: ignore
```

Comment:
I'd ignore the specific codes if possible rather than a blanket ignore: https://mypy.readthedocs.io/en/stable/error_codes.html

What's the reason behind ignoring this one?

Reply:
I ran into `Unsupported right operand type for in ("None")  [operator]` . I'm not sure the best way to address this.

Reply:
Although I'm not sure why I ran into this now given I didn't change the code at all.